src/models/pang15/repairs.md

```diff
@@ -0,0 +1,242 @@
+2. Slowly peel off the black coverings and their connecting strip from the heatsink, highlighted green below.
+
+![Thermal system covers](./img/thermal-system-covers.webp)
+
+3. Remove the six fan screws, highlighted green below.
+
+![Thermal screws](./img/thermal-system.webp)
+
+4. Unplug the fan connectors, highlighted cyan above.
+5. Unscrew the four heatsink screws, highlighted red above.
+    - The heatsink screws are held captive and will not come out of the heatsink.
+6. Remove the fan and heatsink assembly from the system, being careful not to bend the heatsink pipe. It may take some pressure to break the seal of the thermal paste.
+    - The fans are held onto the heatsink with adhesive strips on both the top and bottom; if they come loose, place them back into position between the strips before reinstalling the fan/heatsink assembly.
+7. Using a paper towel, remove the existing silver thermal paste. You may also use a small amount of rubbing alcohol if the old paste is dried or difficult to remove.
+    - The pink thermal pads do not need to be removed.
+
+![Thermal paste removal](./img/thermal-paste-removal.webp)
+
+8. After cleaning the CPU and heatsink, apply a small line of thermal paste directly onto the CPU chip.
+
+![Thermal paste application](./img/thermal-paste-application.webp)
+
+9. Carefully replace the heatsink.
+10. Tighten the four heatsink screws.
+11. Replace the fan screws, plug in the fan connectors, stick the black coverings back onto the heatsink pipes, and replace the bottom panel.
+
+## Replacing the speakers:
+
+The system has four bottom-firing speakers (two tweeters and two woofers). The left and right channels can be replaced independently.
+
+**Part numbers:**
+- Left speakers (right side of machine when viewing from below):
+    - Tweeter: `GS-NM16TG-L`
+    - Woofer: `GS-NM16TG-L-2`
+- Right speakers (left side of machine when viewing from below):
+    - Tweeter: `GS-NM16TG-R`
+    - Woofer: `GS-NM16TG-R-2`
+
+**Tools required:** Cross-head (Phillips) screwdriver    
+**Time estimate:** 10 minutes    
+**Difficulty:** Medium <span style="color:orange;">●</span>
+
+### Steps to replace the right speakers:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Unplug the `SPK1` connector, highlighted cyan below.
+    - Peel back any black tape covering the wires stemming from this connector.
+    - This can alternatively be done after freeing the speakers.
+
+![Right speakers and wire](./img/speakers-right.webp)
+
+3. Unscrew the right tweeter screws, highlighted pink above.
+4. Pull the right woofer off of the plastic mounting posts (highlighted yellow above), then lift the speakers out of the machine.
+    - The woofer wire is not routed underneath the battery by default, but may become routed underneath the battery after the battery is replaced. If this is the case, it may be necessary to [remove the battery](#replacing-the-battery) before removing the right woofer.
```

Comment:
Battery section link is broken. Should be `#removing-the-battery`, looks like.

Reply:
Updated the title for that section instead. I think that was a holdover from copying out of a section for a product with an externally removable battery (where it had to be removed before the bottom panel).